cclib/bridge/cclib2pyscf.py

```diff
@@ -22,12 +24,55 @@ def _check_pyscf(found_pyscf):
 def makepyscf(atomcoords, atomnos, charge=0, mult=1):
     """Create a Pyscf Molecule."""
     _check_pyscf(_found_pyscf)
-    mol = gto.Mole(
-        atom = [['{}'.format(atomnos[i]),atomcoords[i]] for i in range(len(atomcoords))],
-        unit="Angstrom",
-        charge=charge,
-        multiplicity=mult
-    )
-    return  mol
-  
+    # See what can be parsed:
+    inputattrs = data.__dict__
+
+    if hasattr(data, "atomcoords") and hasattr(data, "atomnos"):
+        mol = gto.Mole(
+            atom=[
+                ["{}".format(data.atomnos[i]), atomcoords[i]]
+                for i in range(len(atomcoords))
+            ],
+            unit="Angstrom",
+            charge=charge,
+            multiplicity=mult,
+        )
+
+    return mol
+
+
+def makepyscf_from_molden(data):
+    """
+    Creates an molecule object and additional pyscf data formatted for PySCF.
+    Parameters:
+    ----
+    data: cclib data object from parsed output file
+
+    Returns:
+    ----
+    mol: a pyscf molecule object
+        a molecule object that consists of available data in Molden file.
+    pyscf_data: dictionary
+        a dictionary of additional values returned in an appropriate format for use in PySCF. 
+        If available in ccdata, the following objects are available in the dictionary, otherwise, values are None

```

Comment:
Missing a listing of attributes?

Reply:
Do you mean to define the data types within the dictionary argument?

Reply:
It was more of the docstring implying that there would be some description of the attributes available in the dictionary. But I think I understand on second read: if the right attributes are present in the `ccData` object, the dict will be populated, otherwise it'll be empty (not `None`, or are they empty lists? not clear from the pyscf code).

Actually, rather than going through the MOLDEN writer (which may be lossy), why not copy https://github.com/pyscf/pyscf/blob/7a57d2f37184ea4cd3b52a590c945197b084f66d/pyscf/tools/molden.py#L272 here?

Reply:
That sounds like a a plan. 
I think what will make sense is to first add the ability to the `make_pyscf` function to handle setting up the basis of the molecule if `ccdat['gbasis']` is present, as the basis information is stored within the molecule object. This second function will serve to set up the  MO related internally. 

I'll just need a bit of time to figure out the cleanest way to convert the data to PySCF.